config/generic-mellow-fly-d5.cfg

```diff
@@ -0,0 +1,143 @@
+# This file contains common pin mappings for the Mellow Fly D5
+
+# Important! Do not use this config with a Fly DP5 board!
+
+# To use this board in serial mode, during "make menuconfig", select "Enable
+# low-level configuration options", select the STM32F072
+# micro-controller, select a "8KiB bootloader", and select a "8Mhz
+# crystal", "USB (On PA11/PA12)"
+
+# To use this board in CAN bus mode(maybe along with a CAN bus toolhead
+# board), during "make menuconfig", select "Enable low-level
+# configuration options", select the STM32F072 micro-controller,
+# select a "8KiB bootloader", and select a "8Mhz crystal",
+# "USB to CAN bus bridge (USB on PA11/PA12)"
```

Comment:
It seems to be possible to use this board either in:
* CAN mode - This means the board itself is connected via CAN
* USB-to-CAN Bus Bridge mode - This means the board serves as a CAN bus adapter, e.g. for a toolboard and is itself connected via USB